Dockerfile

```diff
@@ -1,29 +1,39 @@
 FROM pingcap/tidb-enterprise-tools:latest
 
-ARG VERSION=v1.48.0
+ARG RCLONE_VERSION=v1.51.0
```

Comment:
The new version supports [AWS web identity role](https://github.com/rclone/rclone/pull/3837), which can be used in EKS without specifying credentials explicitly.